Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h

```diff
@@ -433,9 +433,9 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRML
   /// Set orientation as 9 values: x, y, and z axis directions, respectively.
   void SetNthControlPointOrientationMatrix(int n, double orientationMatrix[9]);
   /// Set orientation as 9 values: x, y, and z axis directions, respectively.
-  /// Important: this method just calls SetNthControlPointOrientationMatrix and
-  /// does not convert from world coordinate yet.
   void SetNthControlPointOrientationMatrixWorld(int n, double orientationMatrix[9]);
+  /// Get orientation as 9 values: x, y, and z axis directions, respectively.
```

Comment:
Until now, orientation was not tested/incomplete. Now that you fix it, we should do it properly.

The order of elements is not consistent now:
- (x0 x1 x2 y0 y1 y2 z0 z1 z2) order: this is used in the documentation here and in the new function that you added
- (x0 y0 z0 x1 y1 z1 x2 y2 z2) order: this is used [in the schema](https://github.com/Slicer/Slicer/blob/39be4408f084b3e5d632b373a4985cd7fa6c661b/Modules/Loadable/Markups/Resources/Schema/markups-schema-v1.0.2.json#L163) and in [in vtkMatrix3x3](https://github.com/Kitware/VTK/blob/9e838ccf55a67da763399693505907d6ce7222c3/Common/Math/vtkMatrix3x3.cxx#L54-L62)

We don't really have the liberty to choose this, because vtkMatrix3x3 will not change. If we chose a different interpretation then it would lead to errors in the future. So, I think we should standardize on the vtkMatrix3x3 interpretation, even if previous behavior was different.

Reply:
> So, I think we should standardize on the vtkMatrix3x3 interpretation, even if previous behavior was different.

Ok, this addresses my previous question. I will update the class to use the order of elements from VTK.

Reply:
Sounds good. Let me know when it is ready for review.